### DISCUSSION TITLE
Invalid copilot token: missing token: 403

### DISCUSSION THREAD

**Discussion by zlotnik** [community/community#19157] [Copilot Conversations]
I have a message like "Copilot activation failed"

and I have following logs:
[INFO] [auth] [2022-06-22T10:00:36.551Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-06-22T10:00:36.554Z] GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"

I have been already added to pilot groups list and I copilot work for me on 11.06.2022. I am already logged in to git in VS Code.

I've tried to reinstall VS Code Copilot extension few time and log out/log in to github but without success

**Comment by melvincwng** [comment]
@zlotnik, to fix this issue, try the following:

1. Visit https://github.com/github-copilot/tp_signup.
2.  Click on "Continue without payment details" to use for free until August 22.
3. Check if issue is fixed.

Let me know if this helps, thanks

**Comment by rauzerfabiano** [comment]
Hello @zlotnik, try this: 

1. Go to https://github.com/github-copilot/tp_signup
2. Press the button [Continue without payment details.](https://github.com/github-copilot/signup/success)
3. In Suggestions matching public code; select Allow or Block.
4. Press Save and get started
5. Close or Reload the IDE or Code Editor
